### PR TITLE
Steam Api call

### PR DESCRIPTION
**Pull request by ChrisCoffey**
I've created a separate process that makes a Steam API call (right now for some random guy from Des Moins) using the api key @shadow7 provided. This pull is mostly to illustrate how I'm thinking of structuring the actual crawlers. 

For Steam, batches of 100 ids (Steam API limit) will be handed to a child process. The child process will first lookup the steamIds for each Proxy user from Mongo. It will then request the state for each user from Steam & write any changes to the db.

This also has an initial schema (incredibly maleable though) for the `ActivityFeed`, so please look that over and let me know what you think.

### CODE REVIEW

**Comment by shadow7** [comment]
Looks good other than a couple nits. Going forward I'd like to keep variable and function syntax more literal per Donald Knuth's book on literal programming. Lets try to avoid single letter variables even if the code starts to get long.

File: middleware/core/db.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const S = require('./appStates.js');
```
Comment (shadow7) [inline middleware/core/db.js]: appStates or states

File: middleware/core/models.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+var M = require('mongoose');
```
Comment (shadow7) [inline middleware/core/models.js]: mongoose

**Comment by ChrisCoffey** [comment]
Sure, I'll update those. FWIW, Knuth's book has nothing to do with variable or function naming. For namespacing functions, personally I prefer a single or short upper-case identifier, but I'm fine using what you proposed.

**Pull request merged**